duct.go

```diff
@@ -324,7 +356,7 @@ func (c *Composer) Teardown(ctx context.Context) error {
 
 			log.Printf("Removing container: [%s]", cont.Name)
 			if err := client.RemoveContainer(dc.RemoveContainerOptions{ID: cont.id, Force: true, Context: ctx}); err != nil {
-				log.Println(err)
+				log.Printf("Error shutting down continer: [%s] %v\n", cont.Name, err)
```

Comment:
you misspelled container, also please do not add newlines to log messages, they gum up the output.

Reply:
good catch, ya forgot this wasn't regular printf.